lib/ansible/galaxy/role.py

```diff
@@ -311,4 +311,13 @@ def spec(self):
         """
         return dict(scm=self.scm, src=self.src, version=self.version, name=self.name)
 
-
+    @property
+    def version_string(self):
+        """
+        Returns "rolename (version)" if version is not null
+        Returns "rolename" otherwise
+        """
+        if self.version:
+            return "%s (%s)" % (self.name, self.version)
+        else:
```

Comment:
@Lujeni meh, it's really not that important. I prefer it the way it is, and you're entitled to your preference. 

Unless you can point to actual documentation backing up your preferences though (PEP-8, Ansible coding standards, etc.), I'm unlikely to change my code.